plotting_themes.json

```
@@ -0,0 +1,34 @@
+{
+    "dark": {
+        "axes.edgecolor": "white",
+        "axes.labelcolor": "white",
+        "xtick.color": "white",
+        "ytick.color": "white",
+        "legend.framealpha": 0.1,
```

just my taste ignore if disagree
```suggestion
        "legend.framealpha": 0.3,
```

---

I have in general no strong opinions here, I'll go with whatever @AdrianSosic prefers.

---

same, no opinion

---

I'll double check and choose whichever version I prefer ;)